public/react/SettingsSidebar/SettingsSidebar.tsx

```diff
@@ -22,7 +22,12 @@ export interface SettingsSidebarProps {
     overrides: Partial<NormalizeIssueConfig & { fields: string[] }>
   ) => void;
 }
-
+const SETTINGS_WIDTH: Record<string, string> = {
+  SOURCES: "w-96",
```

Comment:
Explicit widths have to be provided for the width transition to work.
It does not work with `w-full` or a custom value `w-[600px]`

Reply:
Let's revisit this with Justin

Reply:
Why is this necessary?  Can't we calculate the width at the time of the animation and temporarily hard-code it?  

We shouldn't have to remember to update these values if the HTML/CSS structure changes in one of the panels.

Reply:
I agree that is not ideal, but if we want to dynamically calculate its width (especially that each setting panel have a different width). The setting panel needs to be rendered first in order to calculate its width, we can not calculate it in advance.

If we do that, this means that first render will be without any animation and then subsequent renders will have access to its width and can apply the width transition.

Or did you mean something else? 

Once I have some downtime on Annuity Association, I can and try and see if there are other better alternatives.

Reply:
Another non-ideal solution is to render an absolutely positioned hidden version of the selected setting panel that we can use to calculate the width in advance